.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_BLOCKS_LENGTH_URL = 'http://127.0.0.1:8000/blocks/length'
+REACT_APP_TRANSACTIONS_LENGTH_URL = 'http://127.0.0.1:8000/transactions/length'
```

Comment:
store only base url and append path in code where it is used. You can use something like `BACKEND_URL` or `SERVICE_NAME_URL` as name.

Reply:
done